samza-yarn/src/main/scala/org/apache/samza/job/yarn/ClientHelper.scala

```diff
@@ -350,6 +350,11 @@ class ClientHelper(conf: Configuration) extends Logging {
       } else {
         false
       }
+    } catch {
+      case e: IOException => {
```

Comment:
Minor: Would prefer to not guess the exception cause in the exception message. Maybe instead add a comment explaining why returning false here is better.

What was the previous behavior? Does the caller catch the exception?